templates/scenarios/ts/command-and-response/teamsfx/app.yml

```diff
@@ -22,10 +22,10 @@ provision:
 deploy:
   - uses: npm/command
     with:
-      args: install
+      args: install --production
```

Comment:
I wonder without devDependency, can it be built successfully? For example, `typescript` is not installed.

Reply:
Good question. Now replaced with npm prune --production.